xml/s4s_tune.xml

```diff
@@ -172,6 +170,21 @@
      <screen>&prompt.root;<command>saptune service takeover</command></screen>
     </step>
    </procedure>
+   <note>
+    <title>Combining optimizations</title>
+    <para>
+      It is possible to combine solutions and notes. However, only one solution can be active at a time. In rare cases, notes can have conflicting options or parameters. To avoid conflicts, arrange your notes, keeping in mind that the last note always overrides conflicting options or parameters of previous notes.
```

Comment:
```suggestion
      It is possible to combine solutions and notes. However, only one solution can be active at a time. In rare cases, notes can have conflicting options or parameters. Arrange your notes carefully to avoid conflicts. The last note always takes priority over conflicting options or parameters of previous notes.
```